maquetado/index.html

```diff
@@ -8,6 +8,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1> 
+  <div class="title"> 
+    <img src= "assets/triangle.svg">
```

Comment:
Include an `alt` property.

Reply:
Images should always have an alt attribute. 
https://blog.hubspot.com/marketing/image-alt-text